jcommune-view/jcommune-web-view/src/test/java/org/jtalks/jcommune/test/utils/Users.groovy

```diff
@@ -20,6 +20,7 @@ import org.jtalks.common.service.security.SecurityContextFacade
 import org.jtalks.jcommune.model.dao.GroupDao
```

Comment:
Do you want to write a component test that checks that users viewing or location service work correctly with sessions?